test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,54 @@
+package util.fixtures;
+
+import entities.Client;
+import enums.Roles;
+import enums.StatusEnum;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public final class ClientServiceFixtures {
```

Comment:
Let's separate this a bit more, create a BaseEntityFixture to fetch entity data with default values and a User fixture fetch user properties and make client call these ones in order to promote code reusability.

Reply:
solved